### ISSUE TITLE
October 2018 Zoe to Tegucigalpa

### ISSUE THREAD

**Issue by juanchon998** [juanchon998/Organization#4]
- [x] Check when fall break is,
- [x] Book plane ticket for zoe to go to tegus!!!!!

**Comment by zoemaisel** [comment]
Fall break is October 6th until October 10th, so we should try to schedule a week to be together

**Comment by juanchon998** [comment]
@zoemaisel October 6th is a Saturday and October 10th is a Tuesday. What say you to taking the rest of the week off to come to Honduras?? Something like arrive in Tegus October 5/6 and leave Tegus on October 13/14?

**Comment by zoemaisel** [comment]
Yes! I'm thinking the same thing...I think planning to arrive in Tegus on the 6th and leaving Tegus on the 13/14 (depending on flight availability) sounds good. 

Shall I start looking into flights?

**Comment by juanchon998** [comment]
Fuckin' do ittttt

**Comment by zoemaisel** [comment]
Proposed option for the [flights](https://goo.gl/flights/FVrf5):
- Sat, Oct 6, 4:30 AM – 9:15 AM (1 stop in SAL  6h 45m  Avianca)
- Sat, Oct 13, 6:30 PM – 3:55 AM+1 (1 stop in SAP  7h 25m  Avianca)

![screen shot 2018-08-21 at 9 02 23 pm](https://user-images.githubusercontent.com/31140352/44436922-a9096a00-a585-11e8-92c6-52e8141956cf.png)
![screen shot 2018-08-21 at 9 02 39 pm](https://user-images.githubusercontent.com/31140352/44436930-aeff4b00-a585-11e8-85fe-c23b6849c214.png)


does this sound goood?

**Comment by zoemaisel** [comment]
Items for @zoemaisel  to bring to Juan in Honduras (use this as a list of things for me to bring to you)!!
- [ ] Kindle and Kindle case!
- [ ] New UNO set for Bayron so he can keep hustling people
- [ ] My watch that Katrinka is finally sending!!!
- [ ] My pot and sweatpants and other shit that my mom shipped over
- [ ] Kindle and Kindle case!!!

**Comment by juanchon998** [comment]
Thank you for the list!!! I'll be sure to add to it 😀 
#fuuuuuuuuuuuuck
![image](https://user-images.githubusercontent.com/27255695/44436543-034cff80-a573-11e8-8863-cdeea915d4b3.png)

**Comment by zoemaisel** [comment]
@juanchon998 peep my added screenshots

**Comment by juanchon998** [comment]
Those look great! This is materializing into a reality :D :D :D :D :D :D :D :D

**Comment by zoemaisel** [comment]
![screen shot 2018-08-23 at 10 59 21 am](https://user-images.githubusercontent.com/31140352/44533572-d0fee780-a6c3-11e8-9f45-f5bdffa51cff.png)
